package/src/lib/SpotlightTour.provider.tsx

```diff
@@ -76,12 +84,13 @@ export interface SpotlightTourProviderProps {
 export const SpotlightTourProvider = forwardRef<SpotlightTour, SpotlightTourProviderProps>((props, ref) => {
   const {
     children,
+    motion = "bounce",
+    nativeDriver = true,
     onBackdropPress,
     overlayColor = "black",
     overlayOpacity = 0.45,
+    spotPadding = 16,
```

Comment:
Is this the default padding of the wrapped component? cause in the documentation it mentions the ```@default``` is 20

Reply:
Yes, that's the actual default. I'll change the docs to match this value. Thanks!